elastic/kibana
# [AO] Add AlertSummaryWidget full-size on the Alerts page

Resolves #148279

## 📝 Summary

Add AlertSummaryWidget full-size on the Alerts page.

![image](https://user-images.githubusercontent.com/12370520/211568369-7d0847ba-e90e-4229-a55a-b68bf55209f8.png)

**Note**
The functional test will be added in another ticket (https://github.com/elastic/kibana/issues/148645)

## 🧪 How to test
- Create a rule that generates an alert and go to the observability > alerts page, you should see the number of alerts correctly in the Alert Summary Widget
- Change the time range to see different formats in the tooltip and the number of buckets will also change accordingly

## 🐛 Known issue
When we don't have any rule, the `_alert_summary` API fails and we show a failed message, I will discuss it with @XavierM, and based on the decision, either I'll update this PR or create a separate PR.
Update: I created a ticket for it (https://github.com/elastic/kibana/issues/148653)
